usaon_benefit_tool/templates/assessment/overview.html

```diff
@@ -27,11 +27,11 @@ <h3>Description</h3>
   <div style="text-align: center;">
 
     <img style="margin: auto" src={{ url_for("legend.get_line_thickness_legend") }} />
-    <p>Criticality rating (thicker line indicates a higher rating)</p>
+    <p>Criticality rating (a thicker line indicates a more critical input, a very thin line is unrated)</p>
 
     <img style="margin: auto" src={{ url_for("legend.get_colorbar_legend") }} />
     <!-- TODO: this label should be baked in to the legend! But was having trouble with the text getting cut off. -->
-    <p>Performance rating</p>
+    <p>Performance rating(grey lines are unrated) </p>
```

Comment:
@rmarow looks like a missing space before the opening paren :)

Reply:
thank you! fixed it :)